tutorials/slicing.adoc

```diff
@@ -1,46 +1,230 @@
+
+Níže uvedeme všechny důležité parametry, které budete při tisku na cvičeních používat. Pokud se chcete něco dozvědět o parametrech, které neuvádíme, doporučujeme pro samostudium vybrat jeden klidný večer nad http://manual.slic3r.org[Slic3r manuálem].
+
+==== Stránka "Layers and perimeters"
+
+Layer height:: Ovlivňuje výšku vrstvy -- přímo ovlivňuje kvalitu a dobu trvání tisku. Rozsah smysluplných hodnot ovlivňuje průměr trysky, která je nainstalovaná na tiskárně.
+
+First layer height:: Ovlivní výšku první vrstvy. Nastavuje se na menší hodnotu než zbytek modelu pro lepší přilnutí k tiskové ploše.
+
+.Porovnání kvality tisku podle výšky vrstvy (https://www.flickr.com/photos/creative_tools/8616820648[Creative Tools, CC BY 2.0])
+image::../images/slicing/8616820648_6e067ceb0f_o.jpg[width=80%, link=../images/slicing/8616820648_6e067ceb0f_o.jpg]
+
+Perimeters:: Počet perimetrů na krajích tisku (neovlivňuje rovné plochy na spodku a vršku předmětu).
+
+Solid layers:: Počet vrstev s plnou výplní na spodku a vršku. Čím více vrstev zvolíme, tím pevnější bude vrchní a spodní povrch a tím lépe se podaří zakrýt nastavený infill. Vysoká čísla však zvýší dobu tisku.
+
+.Jak ovlivňuje tisk nastavení perimetrů
+image::../images/slicing/perimeters.svg[width=80%, link=../images/slicing/perimeters.svg]
+
+==== Stránka "Infill"
+
+Na této stránce lze nastavit vlastnosti infillu, o kterém víme z kapitoly <<Vnitřek modelu>>.
+
+Fill density:: Určuje hustotu vnitřní výplně.
+
+Fill pattern:: Určuje vzorek výplně.
+
+External infill pattern:: Určuje vzorek výplně na vrchních a spodních plochách zvenčí.
+
+Solid infill every:: Zesílí odolnost předmětu vytisknutím plné výplně (100%) každých n vrstev.
+
+Fill angle:: Úhel vzorku výplně.
+
+Solid infill threshold area:: V malých částech předmětu (např. nějaké tenké trubičky či spojovník) slicer vyplňuje plnou výplní (100%). Tato hodnota určuje práh při které se část předmětu považuje jako malá.
+
+.Různé hustoty infillu
+image::../images/slicing/infill.svg[width=80%, link=../images/slicing/infill.svg]
+
+==== Stránka "Skirt"
+
+Na této stránce lze nastavit <<structs, Skirt či Brim>>.
+
+Loops:: Počet obrysů skirtu. Pro běžné použití k vyčištění trysky před tiskem první vrstvy stačí ve většině případů jeden až dva.
+
+Distance from object:: Vzdálenost skirtu od objektu.
+
+Skirt height:: Kolik vrstev bude skirt vysoký. Pro vyčištění trysky se většinou používá pouze jeden.
+
+Exterior brim width:: Nenulová hodnota vygeneruje brim kolem předmětu o zadané velikosti.
+
+==== Stránka "Support material"
+
+Na této stránce lze nastavit <<Supports, supporty>>, nebo <<structs, raft>>.
+
+Generate support material:: Zapne generování podpor.
+
+Overhang threshold:: Určuje hranici, do jak příkrých převisů chcete podpory generovat. Čím vyšší číslo, tím více podpor se vygeneruje.
```

Comment:
Stupně? Odkud měřené?

Reply:
Nezkusíš tu definici přepsat za mě? Asi to nedám tak, aby to bylo pochopitelné, je to docela těžký vysvětlit.

Reply:
```suggestion
Overhang threshold:: Určuje hranici, do jak příkrých převisů chcete podpory generovat. Dá se nastavit v procentech: o kolik procent perimetru ještě smí další vrstva přesahovat vrstvu pod ní (čím vyšší číslo, tím méně podpor); případně ve stupních (náklon od podložky, čím vyšší úhel, tím více podpor).
```